### ISSUE TITLE
Form embed not working

### ISSUE THREAD

**Issue by peterkwells** [OpenAddressesUK/theodolite#201]
@pezholio @Floppy can someone help?

As part of loading up the Error Creative stuff via this PR: https://github.com/OpenAddressesUK/theodolite/pull/199

I inserted the demo form (via instructions at http://embed.openaddressesuk.org/) in a couple of places, namely:

https://github.com/OpenAddressesUK/theodolite/blob/master/app/views/static/services/free-format-address-input.md
https://github.com/OpenAddressesUK/theodolite/blob/master/app/views/static/blog/_posts/2015-05-13-free-format-user-research.md

To support the form I put the embed code in:

https://github.com/OpenAddressesUK/theodolite/blob/master/app/views/shared/_head.html.erb

This worked fine locally but fails in live. I can't see any differences. Can one of you take a look?

**Comment by pezholio** [comment]
Ah, it's because https://alpha.openaddressesuk.org/services/free-format-address-input is loaded over https and the browser is blocking it because it's being loaded from an insecure site. I'll look into it

**Comment by pezholio** [comment]
Yeah, I think this is because we're hosting the the embed code on Github pages. I thought we could have a custom domain with https, but apparently not :disappointed: 

There's two options here:

* Either we switch to using the github.io domain (https://openaddressesuk.github.io/openaddresses_form/), which will work off the bat with https; or
* We host it on Heroku, which will support https (but will cost us an extra $20 a month)

Whaddya reckon?

**Comment by peterkwells** [comment]
if it gives the same functionality/experience to the end-user then .io

Let's not add more costs right now...

**Comment by pezholio** [comment]
Yeah, that was my instinct too. I'll get on it.

**Comment by peterkwells** [comment]
:+1:

**Comment by pezholio** [comment]
Got a PR here https://github.com/OpenAddressesUK/theodolite/pull/202

**Comment by peterkwells** [comment]
sorted

**Comment by peterkwells** [comment]
@pezholio looks like we still have something lingering on this 

1. See: https://twitter.com/OHCamshaft/status/598539396128833537
2. Works fine from my Mac on Safari and Chrome, but fails (404) on my iOs/Safari
